src/dscanner/utils.d

```diff
@@ -128,6 +129,55 @@ string[] expandArgs(string[] args)
 	return rVal;
 }
 
+string absoluteNormalizedPath(in string path)
```

Comment:
make these private or package please, don't want to expose those as D-Scanner APIs.